security/pkg/k8s/tokenreview/k8sauthn.go

```diff
@@ -162,8 +167,41 @@ func (authn *K8sSvcAcctAuthn) ValidateK8sJwt(targetToken, jwtPolicy string) ([]s
 	if err != nil {
 		return nil, fmt.Errorf("unmarshal response body returns an error: %v", err)
 	}
+	return getTokenReviewResult(tokenReview, clusterID)
+}
+
+// isTrustworthyJwt checks if a jwt is a trustworthy jwt type.
+func isTrustworthyJwt(jwt string) (bool, error) {
```

Comment:
I think this function is unchanged, not sure why github show it is

Reply:
yes, unchanged